### DISCUSSION TITLE
How to filter using the `in` operator and `null`?

### DISCUSSION THREAD

**Discussion by moroshko** [prisma/prisma#7882] [Q&A] [answered]
`Food` can have an optional `color`:
```prisma
model Food {
  name  string
  color Color?
}
enum Color {
  RED
  GREEN
  BLUE
}
```
I'd like to get all food with `RED` or `null` color.

I tried this:
```ts
import { Color } from "@prisma/client";

await prisma.food.findMany({
  select: {
    name: true,
    color: true
  },
  where: {
    color: {
      in: [Color.RED, null]
    }
  }
});
```

But, it looks like the `in` operator doesn't accept `null`.

Is there a way to achieve this?

**Comment by ryands17** [comment, marked as answer]
@moroshko 👋 

This should work:

```ts
await prisma.food.findMany({
    where: { OR: [{ color: 'BLUE' }, { color: null }] },
})
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
